SConstruct

```diff
@@ -607,6 +617,7 @@ if PROCEDURAL:
     INSTALL_PROC += env.Install(os.path.join(PREFIX_HEADERS, 'arnold_usd'), ARNOLDUSD_HEADER)
     if env['USD_BUILD_MODE'] == 'static':
         INSTALL_PROC += env.Install(PREFIX_PROCEDURAL, usd_input_resource_folder)
+        INSTALL_PROC += env.Install(PREFIX_PROCEDURAL, usd_input_resource_folder)
```

Comment:
isn't it the same line that is being duplicated ?

Reply:
it is, I didn't notice it, thanks !